### DISCUSSION TITLE
Email notifications for meadia requested by all users

### DISCUSSION THREAD

**Discussion by sean-sullivan1** [sct/overseerr#1083] [Help]
I think I may misunderstand how the notification system works in Overseerr. I have my account with the "owner" role and all permissions enabled. I have enabled email notifications with these settings:
![1](https://user-images.githubusercontent.com/46687337/109974392-83698980-7cc7-11eb-8705-a151d8c09cfc.PNG)
![2](https://user-images.githubusercontent.com/46687337/109974407-8795a700-7cc7-11eb-9419-9e63917aedbc.PNG)

I want to receive notification emails when ANY user requests content that is auto approved. Am I missing something?

**Comment by samwiseg0** [comment]
465 is depreciated. Use 587 without SSL enabled. Also check your logs for errors.

**Reply by sean-sullivan1** [reply]
@samwiseg0 hey I’m still getting emails for the requests I make just not the requests my users make. Modifying smtp settings thanks.

**Comment by TheCatLady** [comment]
Hi @sean-sullivan1!  "Media Approved" e-mails are only sent to the user who submitted the request, as the email is intended to serve as a notification to them that their request was approved.

We are adding a new notification type for admins to receive notifications on automatically approved requests, however.  You can follow #1137 for updates.
